### ISSUE TITLE
[ToolbarButton]: a11y  nvda screen reader does not announce label correctly

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#10562]
### Bug Description

https://sap.github.io/ui5-webcomponents/components/ToolbarButton/
upon pressing Enter on the icon, If accessibleName attribute is provided, nvda announces it twice. first, the value of accessibleName and then default value "Exit Full Screen" which is confusing to the user.
```
<ToolbarButton
  key="exitFullScreen"
  data-testid="exit-full-screen"
  icon={"exit-full-screen"}
  design={ButtonDesign.Transparent}
  onClick={handleExitFullscreenButtonClick}
  ref={exitFullscreenBtnRef}
  accessibleName="Exit Full Screen"
/>
```
Also if you tab to the icon it announces: "Exit Full Screen button"
When you press Enter it announces: "Exit Full Screen"

### Affected Component

ToolbarButton

### Expected Behaviour

When accessibleName attribute has value, nvda should not announce the default value e.g. "Exit Full Screen". Also when user presses Enter on the icon announce 'button' as well after the text. 
Correct behavior is already observed in SAP UI5 here:
https://sapui5.hana.ondemand.com/#/entity/sap.f.FlexibleColumnLayout/sample/sap.f.sample.FlexibleColumnLayoutWithTwoColumnStart

Also if accessibleName is not used nvda should announce the default button name on tabbing. It could be the same it shows on pressing Enter

### Steps to Reproduce

1.
2.
3.
...

### Log Output, Stack Trace or Screenshots

![Image](https://github.com/user-attachments/assets/a5e049ae-5898-4101-a972-8fca1ce9bc4d)

![Image](https://github.com/user-attachments/assets/ac9b58eb-1002-423e-b849-cecb878920ff)

### Priority

Medium

### UI5 Web Components Version

2.x

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @gurkirpalgill ,
I am not able to reproduce this issue.
Please, make a small isolated sample to showcase your problem using https://stackblitz.com/

**Comment by gurkirpalgill** [comment]
Here is the isolated example:
https://stackblitz.com/edit/github-f9pvpx-wrabtmgh?file=src%2FApp.tsx

![Image](https://github.com/user-attachments/assets/39271f14-d597-4c0a-9c1e-b6207fc58d55)

**Comment by IlianaB** [comment]
Hello @SAP/ui5-webcomponents-topic-b , same is reproducible with ui5-button: https://stackblitz.com/edit/github-f9pvpx-gwey11q5?file=src%2FApp.tsx
Upon tabbing or pressing Enter on the button, NVDA announces twice "Exit Full Screen", "Exit Full Screen 1"

**Comment by unazko** [comment]
Hello @gurkirpalgill,

1) For the redundant announcement: 
- My recommendation would be to not provide an aria-label via the `accessibleName` attribute at all:
https://stackblitz.com/edit/github-f9pvpx-tqadnigf?file=src%2FApp.tsx
and just go only with a tooltip.
The tooltip acts both ways - as an accessible name and also it is visually displayed.
That way you would receive a single screen reader announcement.

Previously the screen readers (NVDA, JAWS) were announcing only the aria-label or the tooltip when the they had the same text:
https://jsfiddle.net/unazko/rqydoe5m/7/
Currently with the new screen reader versions that is not the case as they are both announced and there is nothing we can do at our side.

2) For the lacking button type announcement:
- When the button get's activated and changes it's state only the changes should be announced.
That means that the button type would not be announced on activating the button.

Best regards,
Boyan
